src/handlers/homework.ts

```diff
@@ -0,0 +1,44 @@
+// GET
+import { CustomDynamoDB } from "../dynamodb/database";
+import responseHelper from "../helpers/responseHelper";
+import { validateToken } from "../helpers/validations";
+
+const homeworkDB = new CustomDynamoDB('local-bootcamp-classroom-api-homeworks', 'id')
+
+  
+export const handler = async(event: any) => {
+    const tokenData = await validateToken(event.headers.Authorization)
+
+    if(!tokenData){
+        return responseHelper(403, 'Fail Authenticating');
+
+    }
+
+    if(!event.pathParameters || event.pathParameters.homeworkId.trim() === '') {
+        return responseHelper(400, 'Homework ID is required')
+    }
+
+    const homeworkId = event.pathParameters.homeworkId;
+
+    const homework = await homeworkDB.getItem(homeworkId);
+
+    if(!homework) {
+        return responseHelper(400, 'Homework not found')
+    }
+    
+    const homeworkSubmissionDB = new CustomDynamoDB('local-bootcamp-classroom-api-homeworkSubmissions', 'id')
```

Comment:
This database creation should be outside of the function, the same as homeworkDB